x-pack/solutions/security/plugins/elastic_assistant/server/lib/prompt/defend_insight_prompts.ts

```diff
@@ -0,0 +1,83 @@
+- keep processes that reside within the antivirus' main and nested filepaths (e.g., C:\ProgramData\Microsoft\Windows Defender\..., C:\Program Files\AVG\..., C:\Program Files\Avast Software\..., /Applications/AVGAntivirus.app/...)
+- ignore events that are from non-antivirus operating system processes (e.g. C:\Windows\System32\...)
+- ignore events that are single run processes (e.g. installers)
+- ignore events that are from temp directories
+- ignore events that are from Elastic Agent or Elastic Defend
+- group the processes by the antivirus program, keeping track of the agent.id and _id associated to each of the individual events as endpointId and eventId respectively
+- if there are no events, ignore the group field
+- never make any changes to the original file paths
+- new lines must always be escaped with double backslashes, i.e. \\\\n to ensure valid JSON
+- only return JSON output, as described above
+- do not add any additional text to describe your output
+`;
+
+const DEFEND_INSIGHTS_POLICY_RESPONSE_FAILURE_PROMPT = `
+You are a leading expert on resolving Elastic Defend configuration issues. Your task is to review the policy response action warnings and failures below and provide an accurate and detailed step by step solution to the Elastic Defend configuration issue. Organize your response precisely to the following rules:
+- group the policy responses by the policy response action name, message, and os (actions.name:::actions.message:::host.os.name)
+- keep track of the agent.id and _id associated to each of the individual events as endpointId and eventId respectively
+- suggest a remediation action to take for each policy response warning or failure, using the remediationMessage field
+- include a remediation link in the remediationLink field only if one is provided in the context
+- if there are no events, ignore the group field
+- new lines must always be escaped with double backslashes, i.e. \\\\n to ensure valid JSON
+- only return JSON output, as described above
+- do not add any additional text to describe your output
+`;
+
+export const DEFEND_INSIGHTS = {
+  INCOMPATIBLE_ANTIVIRUS: {
+    DEFAULT: DEFEND_INSIGHTS_INCOMPATIBLE_ANTIVIRUS_PROMPT,
+    REFINE: `
+You previously generated the below insights using this prompt: ${DEFEND_INSIGHTS_INCOMPATIBLE_ANTIVIRUS_PROMPT}.
+Double check the generated insights below and make sure it adheres to the rules set in the original prompt, removing events only as necessary to adhere to the original rules. In addition:
+- combine duplicate insights into the same 'group' (e.g. AVG + AVG Free + AVG Hub + AVG Antivirus)
+- remove insights with no events
+    `,
+    CONTINUE: `Continue exactly where you left off in the JSON output below, generating only the additional JSON output when it's required to complete your work. The additional JSON output MUST ALWAYS follow these rules:
```

Comment:
Hey I'm updating the prompts for something else and I think you may have forgotten to update the integration when you made these changes. [My PR ](https://github.com/elastic/integrations/pull/15213)will include your changes, so no action needed, but please remember to update the integration in the future